flybywiresim/a32nx
# fix: MODE SEL manual visual improvements

Fixes #4034 

## Summary of Changes

- When MOD SEL is set to MAN: 
          - Removed LDG ELEV info from ECAM Press
          - Removed SYS1/2 display on ECAM Press
          - Removed LDG ELEV info from ECAM Cruise
- When MOD SEL is cycled (set to MAN for at least 10s and then reset to auto) the active cabin pressure control system is switched (SYS1 <-> SYS2).

## Screenshots (if necessary)


Before:
![image](https://user-images.githubusercontent.com/60938089/111850591-9f486e80-8908-11eb-93e1-0a73d1a90cd4.png)

After:
![image](https://user-images.githubusercontent.com/60938089/111850611-b0917b00-8908-11eb-8d6b-0ac4501604c4.png)
![image](https://user-images.githubusercontent.com/60938089/111850627-ba1ae300-8908-11eb-8c70-44227c2eb9f6.png)


## References


![image](https://user-images.githubusercontent.com/60938089/111850797-21389780-8909-11eb-814b-cb7c278a4807.png)
![image](https://user-images.githubusercontent.com/60938089/111850815-2b5a9600-8909-11eb-9395-71250afef416.png)
![image](https://user-images.githubusercontent.com/60938089/111850827-39a8b200-8909-11eb-9da5-f4803b193030.png)
![image](https://user-images.githubusercontent.com/60938089/111850845-488f6480-8909-11eb-9f48-e2d9c3306c2e.png)
![image](https://user-images.githubusercontent.com/60938089/111851412-f2232580-890a-11eb-9126-541968db032f.png)
![image](https://user-images.githubusercontent.com/60938089/111851421-fcddba80-890a-11eb-9809-e16713b777aa.png)





## Additional context

My first pull request :)

Limitations: 
          - The system switch will only happen if the ECAM PRESS page is displayed. This should be better implemented in a future PR to improve the representation of the press system as a whole.
          - The dials and values on the PRESS page point to simvars that are not currently being set anywhere. This I believe was intentional in PR #1373 awaiting the wasm system.


Discord username (if different from GitHub): Commander Ga#1548

## Testing instructions

1. Spawn in any state
2. Set the MODE SEL pushbutton in the CABIN PRESS overhead panel to manual
3. Observe the CRUISE page and PRESS page of the ECAM and verify no LDG ELEV information shows
4. Set the MODE SEL back to auto, if it's within 10s of the first press, the SYS showing on the PRESS page should stay the same, if it's after 10s, it should switch to the other SYS.
5. Verify no other issues appear with any of the other ECAM pages in any stages of flight.



## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
